routes/index.js

```diff
@@ -1,5 +1,10 @@
 var express = require('express');
+var clientID = "692884782115-u4o2n8dco40hjqa18b1agl9492m05l1j.apps.googleusercontent.com";
+var GoogleAuth = require('google-auth-library');
+var auth = new GoogleAuth;
```

Comment:
Can you use parentheses? `new GoogleAuth();`

For more style things, consider running this through JSHint: http://jshint.com/